_field-types/supported-field-types/star-tree.md

```diff
@@ -105,17 +105,20 @@ PUT logs
 }
 ```
 
+## STIX configuration options
 
+You can customize your STIX implementation using the following `config` options.
 
-## Star-tree mapping parameters
+| Parameter  | Description   | 
+| :--- | :--- |
+| `mappings` | A list of [mapping](#mapping-parameters) parameters. Required.
+| `max_leaf_docs` | The maximum number of STIX documents that a leaf node can point to. After the maximum number of documents is reached, children nodes will be created based on the unique vales of the next field in the `ordered_dimension` (if any). Default is `10000`. A lower value will use more storage but result in faster query performance. Inversely, a higher value will use less storage but result in slower query performance. For more information, see [Star-tree indexing structure]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index/#star-tree-index-structure).  |
+| `skip_star_node_creation_for_dimensions`  | A list of dimensions for which STIX will skip star node creation. When `true`, this reduces storage size at the expense of query performance. Default is `false`. For more information about star nodes, see [Star-tree indexing structure]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index/#star-tree-index-structure). |
 
-Specify any star-tree configuration mapping options in the `config` section. Parameters cannot be modified without reindexing documents.
+### Mapping parameters
```

Comment:
Same comment as above, these are all part of config. Lets not mention `mappings` . Please refer the associated json and we can word it on how they're structured

Reply:
Removed this section.